zephyr/src/supp_api.c

```diff
@@ -141,13 +116,12 @@ static inline void wpa_supp_restart_status_work(void)
 }
 
 
-int zephyr_supp_connect(const struct device *dev,
+int z_wpa_supplicant_connect(const struct device *dev,
```

Comment:
is there a corresponding change in sdk-zephyr?

Reply:
Yes, the `sdk-nrf` PR linked has the manifest change.

Reply:
:+1: